src/app/clusters/ota-requestor/DefaultOTARequestor.cpp

```diff
@@ -183,8 +183,8 @@ void DefaultOTARequestor::OnQueryImageResponse(void * context, const QueryImageR
 
         if (update.softwareVersion > requestorCore->mCurrentVersion)
         {
-            ChipLogDetail(SoftwareUpdate, "Update available from version %" PRIu32 " to %" PRIu32, requestorCore->mCurrentVersion,
-                          update.softwareVersion);
+            ChipLogProgress(SoftwareUpdate, "Update available from version %" PRIu32 " to %" PRIu32, requestorCore->mCurrentVersion,
```

Comment:
Blocking comment since I am still making sure that automated testing does in fact need this